src/generate.ts

```diff
@@ -152,6 +154,13 @@ const getRouteSummary = (ctx: Context, symbol: ts.Symbol): string | undefined =>
     .map((tag) => tag.text)
     .filter(isDefined)[0]
 
+const getRouteTags = (ctx: Context, symbol: ts.Symbol): string[] | undefined =>
+  symbol
+    .getJsDocTags()
+    .filter((tag) => tag.name === 'tags')
+    .map((tag) => tag.text?.split(','))
+    .filter(isDefined)[0]
```

Comment:
```suggestion
    .filter(isDefined)
    .flatten()
    .map(tag => tag.strip())
```
I think this would support multiple tags more nicely